opendatahub-io/odh-dashboard
# Add incubation overlay

## Description


Closes #1614 

kfdef

```yaml
apiVersion: kfdef.apps.kubeflow.org/v1
kind: KfDef
metadata:
  name: odh-dashboard
spec:
  applications:
    - kustomizeConfig:
        overlays:
          - incubation
        repoRef:
          name: manifests
          path: manifests
      name: odh-dashboard
  repos:
    - name: manifests
      uri: >-
        https://github.com/lucferbux/odh-dashboard/tarball/add-incubation-overlay
```

## How Has This Been Tested?



1. Add the following kfdef
2. Check that `odh-dashboard` deployment contains `quay.io/opendatahub/odh-dashboard:nightly` as the base image

## Test Impact



## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
